acceptance/bundle/templates/default-python/integration_classic/output.txt

```diff
@@ -217,16 +217,16 @@ Validation OK!
            }
          ],
 -        "name": "[dev [USERNAME]] project_name_[UNIQUE_NAME]_pipeline",
-+        "name": "project_name_[UNIQUE_NAME]_pipeline",
-         "permissions": [],
 -        "schema": "project_name_[UNIQUE_NAME]_dev",
 -        "tags": {
 -          "dev": "[USERNAME]"
 -        }
++        "name": "project_name_[UNIQUE_NAME]_pipeline",
++        "permissions": [],
```

Comment:
The + suggests it includes an empty list here?

Reply:
yes, but it probably included it before. However, it was removed in the file it compares against, so now it shows up in diff.